assignments/ShapeIndexing/scriptClassification.sh

```diff
@@ -11,7 +11,7 @@ NBIMGTESTS=40
 NBTESTS=5
 
 #No noise here
-MAXNOISE=0.0
+MAXNOISE=0.5
```

Comment:
In my last PR I updated the classification script, and as MAXNOISE was fixed to 0, I removed the noise :/ So it's a bug from me...